File: Python/Arrays/09.PairSum.py
from array import * 

def pairSum(arr, brr):
    sum = 12

    for i in range (len(arr)):
        for j in range (len(brr)):
            if arr[i] + brr[j] == sum:
                print("Pair is ", arr[i], " ", brr[j])
                break

arr1 = array('i', [2, 3, 6, 8, 1, 9, 3])
arr2 = array('i', [4, 6, 9, 3, 2, 1, 7])

pairSum(arr1, arr2)

File: CPP/Arrays/09.PairSum.cpp
#include <bits/stdc++.h> 
using namespace std;


void pairSum(vector<int> arr, vector<int> brr){
    int val = 12;
    
    for (int i =0; i< arr.size(); i++){
        for (int j = 0; j< brr.size(); j++){
            if (arr[i] + brr[j] == val){
                cout<<"Pair is "<<arr[i]<<", "<<brr[j]<<endl;
            }
        }
    }
}


int main()
{
    vector<int> arr = {1, 3, 4, 6, 9, 7};
    vector<int> brr = {3, 5, 6, 4, 2};
    
    pairSum(arr, brr);

    return 0;
}
